### DISCUSSION TITLE
Score data very behind live games

### DISCUSSION THREAD

**Discussion by zgall111** [riffnshred/nhl-led-scoreboard#370] [General]
I managed to get the scoreboard up and running on a Pi Zero (I know you warned against but it's all I had). It seems to be running as expected except the live scores are very far behind the actual games. I can't imagine this would be related to the Pi Zero but I obviously could be wrong. Have there been any other recent reports of scores that are slow to update?

**Comment by riffnshred** [comment]
Hey man, glad you got it up and running. No report of such these days. The NHL API can be off sometimes and our theory is that they let some drunk intern take control of it. To be sure the scoreboard match the source of data (not the score on TV), go on nhl.com and look at the score there. 

Sadly I think your issue is caused by the fact that you use a Raspberry Pi Zero. The screen rendering takes the vast majority of the single CPU core to do its job and only a really small amount is left to do the data gathering and parsing and managing the state loops etc... 

If you are in the US, I hear microcenter has some raspberry pi in stock sometimes, you might get lucky there.

**Comment by zgall111** [comment]
I was hoping you weren’t going to say that but I guess it makes sense.

Would a Pi Zero 2W work?

**Comment by riffnshred** [comment]
Its has the same CPU than the Raspberry pi 3B+.  Its a 4 core CPU which is plenty of power for this project.
